pkg/csi/mocks/mock_snapshot_creator.go

```diff
@@ -8,8 +8,8 @@ import (
 	context "context"
 	gomock "github.com/golang/mock/gomock"
 	snapshot "github.com/kanisterio/kanister/pkg/kube/snapshot"
-	v1alpha1 "github.com/kanisterio/kanister/pkg/kube/snapshot/apis/v1alpha1"
 	types "github.com/kastenhq/kubestr/pkg/csi/types"
+	snapv1 "github.com/kubernetes-csi/external-snapshotter/client/v4/apis/volumesnapshot/v1"
```

Comment:
@PrasadG193 Just wondering if you had manually edited these files or if you ran `go generate` at the top level.

Reply:
I had updated it manually. Updated PR by running `go generate`. Thanks